frontend/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@spree/frontend",
+  "version": "4.3.0-alpha",
+  "license": "BSD-3",
+  "source": "app/javascript/index.js",
+  "main": "dist/spree_frontend.js",
+  "exports": "dist/spree_frontend.modern.js",
+  "module": "dist/spree_frontend.module.js",
```

Comment:
Webpack can't generate ES6 modules: https://github.com/webpack/webpack/issues/2933#issuecomment-774253975. For `frontend/dist/spree_frontend.module.js` to work, the Spree SDK would have to use Rollup instead of Webpack. `dist/server/index.js` and `dist/client/index.js` are exported in the UMD format. Most importantly, these exports support importing the SpreeSDK via CommonJS in Node and in the browser and referencing it by `window.SpreeSDK` in the browser.

Reply:
https://github.com/spree/spree/pull/10982